pkg/api/admin/dao/repositories/namespace.go

```diff
@@ -51,3 +53,12 @@ func (r NamespaceRepository) GetByCode(ctx context.Context, code string) (*model
 	}
 	return &namespace, nil
 }
+
+// ListNamespaces returns a list of all namespaces.
+func (r NamespaceRepository) ListNamespaces(ctx context.Context) ([]models.Namespace, error) {
```

Comment:
What do we think of the stuttering implied by this naming convention, ie "namespaces" appearing twice: namespaceRepository.ListNamespaces. Maybe #List would be clear enough and shorter?

Reply:
Yep!